tutor/src/screens/question-library/dashboard.jsx

```diff
@@ -37,11 +69,18 @@ class QuestionsDashboard extends React.Component {
   @action.bound onSelectionsChange(pageIds) {
       this.pageIds = pageIds;
       this.isShowingExercises = !isEmpty(pageIds);
+      this.setPageIdsQuery(pageIds)
   }
 
   @action.bound onSelectSections() {
       this.showingDetails = false;
       this.isShowingExercises = false;
+      this.setPageIdsQuery()
+  }
+
+  @action.bound setPageIdsQuery(pageIds = []) {
+      const query = extend(Router.currentQuery(this.windowImpl), { pageIds: pageIds.join(',') })
+      this.props.history.push(this.windowImpl.location.pathname + '?' + qs.stringify(query))
```

Comment:
we probably want to use `history.replace` here, otherwise we'll create multiple entries in history and the user will have to hit back twice or more to get back to the previous page.